src/main/main.cpp

```diff
@@ -114,6 +127,55 @@ void statusUpdate()
 	}
 }
 
+
+/**
+ * This method will cause buzzer to make a short sound
+ */
+void buzz() {
+	ledcWrite(channel, dutyCycle);
+	delay(300);
+	ledcWrite(channel, 0);
+}
+
+/**
+ * This will make a noise if last noise was less than 30 seconds ago
+ */
+void audibleAmberAlert() {
+	if (timeDiff(alertMillis, AMBER_ALERT_SOUND_DELAY)) {
+		alertMillis = millis();
+		Serial.println("Amber buzz");
+		buzz();
+	}
+}
+
+/**
+ * This will make a noise if last noise was less than 5 seconds ago
+ */
+void audibleRedAlert() {
+	if (timeDiff(alertMillis, RED_ALERT_SOUND_DELAY)) {
+		alertMillis = millis();
+		Serial.println("Red buzz");
+		buzz();
+	}
+}
+
+/**
+ * Check if it should call method to make noise
+ */
+void whichAlertToMake() {
+	if (pir->getPIRStatus() == "OCCUPIED") {
+		if (thermometer->getTempStatus(tempStatus) == "RED"
+				|| (humidity->getHumStatus(humStatus)) == "RED") {
+			audibleRedAlert();
+		} else if (thermometer->getTempStatus(tempStatus) == "AMBER"
+				|| (humidity->getHumStatus(humStatus)) == "AMBER") {
+				
+			audibleAmberAlert();
+		}
+	}
+	ledcWrite(channel, 0);
+}
+
```

Comment:
Is it possible to put these methods in their own class?

Reply:
Now in own class